plugins/main/public/react-services/navigation-service.tsx

```diff
@@ -124,6 +124,34 @@ class NavigationService {
     return getCore().application.getUrlForApp(appId, options);
   }
 
+  public switchTab(newTab: any, force: any) {
```

Comment:
`newTab` is supposed to be of type string and force I understand is a `boolean`, although `force` is not necessary there since it is not used. Right?
It's a minor change but it can prevent errors from passing incorrect parameters.

Reply:
Resolved in this [commit]([449abd7](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6775/commits/449abd7f000be516ccb589d78c6b1af7348dd746))